.github/actions/invalidate_master_cache/action.yml

```diff
@@ -0,0 +1,11 @@
+name: invalidate_npm_cache
```

Comment:
As far as I checked you didn't use this anywhere.
You need to save the cache from master branch

Reply:
its now on `release_staging` workflow